src/IbuMethods.cpp

```diff
@@ -61,3 +67,49 @@ double IbuMethods::rager(double AArating, double hops_grams, double finalVolume_
 
    return (hops_grams*utilization*AArating*1000)/(finalVolume_liters*(1+gravityFactor));
 }
+
+
+/*!
+ * \author Daniel Pettersson (pettson81@gmail.com)
+ *
+ * \brief Calculates the IBU by Greg Noonans formula
+ */
+double IbuMethods::noonan(double AArating, double hops_grams, double finalVolume_liters, double wort_grav, double minutes)
+{
+    double gramsPerOz = 28.3495231;
+    double litersPerFiveGallons = 18.9270589; // US gallons
+    double volumeFactor = litersPerFiveGallons / finalVolume_liters;
+    double hopsFactor = hops_grams/ gramsPerOz;
+
+    //using 60 minutes as a general table
+    double utilizationFactorTable[4][2] =  {
+                     {1.050, 1},
+                     {1.065, 0.9286},
+                     {1.085, 0.8571},
+                     {1.100, 0.75}
+                    };
+
+    double utilizationFactor;
+
+    if(wort_grav <= utilizationFactorTable[0][0])
+    {
+        utilizationFactor = utilizationFactorTable[0][1];
+    }
+    else if(wort_grav <= utilizationFactorTable[1][0])
+    {
+        utilizationFactor = utilizationFactorTable[1][1];
+    }
+    else if(wort_grav <= utilizationFactorTable[2][0])
+    {
+        utilizationFactor = utilizationFactorTable[2][1];
+    }
+    else
+    {
+        utilizationFactor = utilizationFactorTable[3][1];
+    }
+
+    double fittedCurve = -0.00000000008229488217 * pow(minutes, 7) +0.00000002132644293 * pow(minutes, 6) - 0.000002102006144 * pow(minutes, 5) + 0.00009925450081 * pow(minutes, 4) - 0.002340415323 * pow(minutes, 3) + 0.02720809386 * pow(minutes, 2) -0.08868853463*minutes + 0.7000029428;
```

Comment:
Use the Polynomial class in Algorithms.h to evaluate polynomials. You can make a static one for this method.